src/phoenix/trace/llama_index/callback.py

```diff
@@ -176,6 +177,28 @@ def __init__(
         self._tracer = Tracer(on_append=callback, exporter=exporter or HttpExporter())
         self._event_id_to_event_data: EventData = defaultdict(lambda: CBEventData())
 
+    def _null_fallback(self, *args: Any, **kwargs: Any) -> None:
+        msg = (
+            "OpenInferenceCallbackHandler trace processing failed, more info is logged to the root"
+            "logger"
+        )
+        logger.exception(msg)
```

Comment:
I think `logger.exception` is intended to be called from within an exception clause. https://docs.python.org/3/library/logging.html#logging.Logger.exception

Did you mean `logger.error`?

Reply:
Nit: We wind up logging two exceptions/ errors instead of one. Can we condense into a single error message?

Reply:
I don't think the second log is required, it's currently a matter of expedience since that's what we were already doing. We could simply remove the second log or rely on the fallback methods to log (but I think I'd prefer any kind of broad error suppression to automatically produce logs)

Reply:
Using the fallback methods to handle errors makes the most sense to me. I would expect the method to not log anything unless I configured it to do so, I don't hold a strong opinion there if you think it should have a default behavior.